src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -731,12 +731,26 @@ class PolicyHandler : public PolicyHandlerInterface,
   void GetRegisteredLinks(std::map<std::string, std::string>& out_links) const;
 
  private:
-  mutable sync_primitives::RWLock policy_manager_lock_;
+  mutable sync_primitives::RecursiveLock policy_manager_lock_;
   std::shared_ptr<PolicyManager> policy_manager_;
   void* dl_handle_;
   std::shared_ptr<PolicyEventObserver> event_observer_;
   uint32_t last_activated_app_id_;
 
+  /**
+   * @brief Call PolicyManager function with sync_primitives::AutoLock
+   * @param func function PolicyManager to call
+   * @param args argmunets for call function
```

Comment:
@SNiukalov typo

Reply:
@AByzhynar 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/2614/commits/6117bd64332230467f2b4e885a9cb9ed05818c4c